### ISSUE TITLE
Unwanted space at the top of DayView when used inside a Sliver.

### ISSUE THREAD

**Issue by mridboy96** [SimformSolutionsPvtLtd/flutter_calendar_view#225]
I have been trying to implement the calendar_view widget in a customScrollView, but for some reason there seems to an unwanted space at the top of the DayView widget.
I am currently using the DayView Widget inside a SliverFillRemaining widget.
I have attached the screenshot and my code below. 
Would like to find some help here as to how I can eliminate this extra space that is present above the DayView Widget.

Code Screenshot
```
Widget build(BuildContext context) {
    return Scaffold(
      body: Stack(
        children: [
          CustomScrollView(
            controller: _scrollController,
            physics: NeverScrollableScrollPhysics(),
            slivers: [
              SliverAppBarWidget(
                isAppBarCollapsed: bloc.isAppBarCollapsed,
                activeUserMonogramText: 'AA',
                staticAppBarTitle: 'Schedule',
                actionWidget: InkWell(
                  borderRadius: BorderRadius.circular(20),
                  onTap: () => bloc.openDatePicker(
                      context: context, dateTime: DateTime.now(), dayViewState: calendarViewKey.currentState),
                  splashColor: AppColors.primary.withOpacity(0.5),
                  child: Container(
                    width: MediaQuery.of(context).size.height <= 812 ? 20 : 25,
                    height: MediaQuery.of(context).size.height <= 812 ? 20 : 25,
                    decoration: BoxDecoration(
                      color: AppColors.primarySwatch.shade100,
                      shape: BoxShape.circle,
                    ),
                    child: Center(
                        child: Icon(Icons.calendar_month_sharp,
                            color: AppColors.primary, size: MediaQuery.of(context).size.height <= 812 ? 15 : 17)),
                  ),
                ),
              ),
              SliverFillRemaining(
                child: DayView(
                  controller: _jobCalendarEventController,
                  key: calendarViewKey,
                ),
              )
            ],
          ),
        ],
      ),
    );
```
![Simulator Screenshot - iPhone 14 Pro - 2023-05-10 at 00 19 46](https://github.com/SimformSolutionsPvtLtd/flutter_calendar_view/assets/38466433/0ab31a84-ad3f-4bb0-9ff9-3f8b1215f0b7)

Preview Screenshot

**Comment by jaiminrana05** [comment]
Hi @mridboy96, could you kindly share your code snippet for the SliverAppBarWidget?

**Comment by mridboy96** [comment]
HI !!
It had been quite a while since I had posted this question.
Ever since then I was able to figure out that in the much later version of the package there is a new option for the DayView widget called ```safeAreaOption: SafeAreaOption(bottom: false, top: false)```
With this specific option I have been able to remove the extra unwanted space that I was initially complaining about
However if I use this specific option, there is some unwanted space that is coming up between the day view and the timeline.
Please see the screenshot for more details.
Would be really helpful if I could get some insights on what is going on and how this can be removed.
Thanks !
![Simulator Screenshot - iPhone 11 - 2023-05-16 at 13 04 50](https://github.com/SimformSolutionsPvtLtd/flutter_calendar_view/assets/38466433/cadc4d07-ceca-46ba-9b65-ce4dfc0b6a40)

**Comment by jaiminrana05** [comment]
Greetings, @mridboy96. It's wonderful news that your problem has been partially resolved. Would you mind sharing your code snippet related to the issue mentioned above?

**Comment by DhavalRKansara** [comment]
Hi @mridboy96  We have fixed the issues in the #251 you will find the latest fixes in our latest release 1.0.4

**Comment by mridboy96** [comment]
That is great !! Thank you!!
I will pick up the PR #251 and check it out and will confirm here !
